src/_data/schemas/lpa.json

```diff
@@ -0,0 +1,289 @@
+{
+    "$schema": "http://json-schema.org/schema",
+    "$ref": "#/definitions/LPA",
+    "definitions": {
+        "LPA": {
+            "type": "object",
+            "description": "The main schema for the Lasting Power of Attorney Document.",
+            "additionalProperties": false,
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "format": "integer",
+                    "description": "The LPA Documents unique identifier.",
+                    "title": "The unique LPA identifier"
+                },
+                "type": {
+                    "type": "string",
+                    "format": "{health_welfare|property_finance}",
+                    "description": "Is this a Health & Welfare or Property & Finance LPA.",
+                    "title": "LPA Type (Health & Welfare or Property & Finance LPA)"
+                },
+                "preferences": {
```

Comment:
Why do we have a maximum character length on preferences and instructions? I imagine some may be quite involved.